articles/fleet-4.50.0.md

```diff
@@ -0,0 +1,137 @@
+# Fleet 4.50.0 | Security agent deployment, AI descriptions, and Mac Admins SOFA support.
+
+![Fleet 4.50.0](../website/assets/images/articles/fleet-4.50.0-1600x900@2x.png)
+
+Fleet 4.50.0 is live. Check out the full [changelog](https://github.com/fleetdm/fleet/releases/tag/fleet-v4.50.0) or continue reading to get the highlights.
+For upgrade instructions, see our [upgrade guide](https://fleetdm.com/docs/deploying/upgrading-fleet) in the Fleet docs.
+
+## Highlights
+
+* Deploy security agents to macOS, Windows, and Linux
+* Policy description and resolutions aided by AI
+* Mac Admins SOFA support
+* `zsh` support
+
+
+## Deploy security agents to macOS, Windows, and Linux
+
+Fleet enhances the deployment capabilities for IT administrators, particularly concerning security agents. Now available in Fleet Premium, this feature allows administrators to add and deploy security agents directly to macOS, Windows, and Linux hosts through the Software page, the Fleet API, or via GitOps workflows. This deployment functionality requires that the host has a `fleetd` agent with scripts enabled, but notably, it does not necessitate MDM (Mobile Device Management) features to be enabled within Fleet. This new capability supports a more streamlined and efficient approach to enhancing host security across diverse operating environments, allowing IT and security teams to ensure their hosts are protected with the necessary security tools without the complexity of additional infrastructure changes.
+
```

Comment:
```suggestion

For users that host Fleet themselves, this feature requires hooking up Fleet with an S3 bucket. See how in the server configuration reference [here](https://fleetdm.com/docs/configuration/fleet-server-configuration#s-3).
```

Suggest adding a not about the new infra requirement: S3

Note that the doc link is currently broken however it will be updated as part of this PR: https://github.com/fleetdm/fleet/pull/19236